### PR TITLE
Pages routes

### PR DESCRIPTION
**Pull request by deekshithmd**
Added pages and routes to the quiz app. Please review it and provide your comments

### CODE REVIEW

**Review by sanjitsarkar — APPROVED** [review]
Code looks clean and easy to understand

**Review by ShrutiLakhani — APPROVED** [review]
Your code looks clean. Couldn't find anything that require changes. Good work!

File: src/components/Quiz/quiz.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.button-container {
+    margin-top: 2rem;
+    display: flex;
+    justify-content: space-between;
+}
+
+input[type="radio"] {
+    appearance: none;
+    width: 20px;
+    height: 20px;
```
Comment (mayuresh-tech) [inline src/components/Quiz/quiz.css:37]: You can try using rem instead of px

**Review by mayuresh-tech — APPROVED** [review]
Good to merge!

File: src/components/Quiz/quiz.css (hunk 1 above)
Comment (deekshithmd) [reply src/components/Quiz/quiz.css:37]: ok, Thank you for pointing it out

**Pull request merged**